packages/kotti-ui/source/kotti-table/KtTable.vue

```diff
@@ -80,7 +81,10 @@ export default {
 		emptyText: { default: 'No Data', type: String },
 		id: { default: null, type: String },
 		// FIXME: Props should either be required or have a default
```

Comment:
Is this comment still relevant?

Reply:
Yes, since there is no `default` provided